src/diagonal.works/b6/api/functions/search.go

```diff
@@ -180,7 +180,7 @@ func keyed(context *api.Context, key string) (b6.Query, error) {
 
 // Wrap a query to only match features with the given feature type.
 func typed(context *api.Context, typ string, q b6.Query) (b6.Query, error) {
-	return b6.Typed{Type: b6.FeatureTypePath, Query: q}, nil
+	return b6.Typed{Type: b6.FeatureTypeFromString(typ), Query: q}, nil
```

Comment:
test passes w/o this 🫠

Reply:
ok it goes through https://github.com/diagonalworks/diagonal-b6/blob/84c275ad54f0da5dcb11cc819dbfaedf55cb5864/src/diagonal.works/b6/api/shell.go#L740

Reply:
idk how to test this tho, lilke i put out typed in a separate line, but it doesnt get evaluated i think ? and when it does / inside find / then it s the same flow via shell

Reply:
I added a test for this, and fixed an underlying issue (with typed query -> proto conversion) that it revealed.

Reply:
tyyyyy